commitlint.config.js

```diff
@@ -1,6 +1,26 @@
 module.exports = {
   extends: ['@commitlint/config-conventional'],
   rules: {
-      'type-enum': [2, 'always', ['build', 'ui', 'feature', 'ci', 'devop', 'chore', 'docs', 'feat', 'fix', 'refactor', 'revert', 'style', 'lint', 'test', 'translations']]
+    'type-enum': [
+      2,
+      'always',
+      [
+        'build',
+        'ui',
+        'feature',
+        'ci',
+        'devop',
+        'chore',
+        'docs',
+        'feat',
+        'fix',
+        'refactor',
+        'revert',
+        'style',
+        'lint',
+        'test',
+        'translation'
```

Comment:
and `sentry` here then edit the devmoji.config.js to add a gitmoji. I used this to get gitmoji names https://gitmoji.dev/